tests/queries/0_stateless/02598_clickhouse_help.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+# Tags: no-parallel
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+
+# We have to use fixed terminal width. It may break other tests results formatting.
+
+backup_stty_size=$(stty size | awk '{print $2}')
+stty columns 120
```

Comment:
Also, there is no tty in CI,
so it's better to set it to the same value used for a fallback when there is no tty.